lib/fennec.ex

```diff
@@ -1,6 +1,19 @@
 defmodule Fennec do
   @moduledoc """
-  STUN/TURN server
+  STUN/TURN servers application
+
+  Fennec allows you to start multiple listeners (later called "servers")
+  which react to STUN/TURN messages. The only difference between servers
+  is a transport protocol and interface/port pair which server uses for
+  listening to STUN packets.
+
+  Each server is independent, i.e. if one of them crashes other should not
```

Comment:
the other one should not ... the recommended method... depend on a server's supervisor configuration.